CSX Trigger tests failing

Getting this error in our trigger tests for CSX now

`[6584] [2023-04-27T07:47:34.601Z] The 'ProductsTrigger' function is in error: Microsoft.CodeAnalysis.Scripting.CompilationErrorException: Script compilation failed.`

```
[10936] [2023-04-27T07:47:43.066Z] You may be referencing NuGet packages incorrectly. Learn more: https://go.microsoft.com/fwlink/?linkid=2091419
[10936] [2023-04-27T07:47:43.086Z] run.csx(1,7): error CS1504: Source file '../../Common/product.csx' could not be opened -- Could not find file.
[10936] [2023-04-27T07:47:43.087Z] run.csx(11,48): error CS0246: The type or namespace name 'Product' could not be found (are you missing a using directive or an assembly reference?)
[10936] [2023-04-27T07:47:43.101Z] Function compilation error
[10936] [2023-04-27T07:47:43.101Z] Microsoft.Azure.WebJobs.Script: Script compilation failed.
[10936] [2023-04-27T07:47:43.103Z] run.csx(1,7): error CS1504: Source file '../../Common/product.csx' could not be opened -- Could not find file.
[10936] [2023-04-27T07:47:43.103Z] run.csx(11,48): error CS0246: The type or namespace name 'Product' could not be found (are you missing a using directive or an assembly reference?)
```

--- a/samples/samples-csx/ProductsTrigger/run.csx
+++ b/samples/samples-csx/ProductsTrigger/run.csx
@@ -1,4 +1,4 @@
-#load "../../Common/product.csx"
+#load "../Common/product.csx"
 #r "Newtonsoft.Json"
 #r "Microsoft.Azure.WebJobs.Extensions.Sql"
 

